thats-the-way.py

```diff
@@ -6,19 +6,21 @@
 """
 
 
-# This function gets a path to a directory and returns a list of the files that start
-# with the word 'deep' in the given directory
 def get_deep_files(path):
     """
+    This function gets a path to a directory and returns a list of the files that start
+    with the word 'deep' in the given directory.
     :param path: a path to the directory we would like to check
     :return: a list of names of the files that start with the word 'deep'
     """
-    my_list = list()
-    for x in os.listdir(path):
-        if x.startswith("deep"):
-            my_list.append(x)
+    files_start_with_deep_list = list()
+    for file_name in os.listdir(path):
+        if file_name.startswith("deep"):
+            files_start_with_deep_list.append(file_name)
```

Comment:
rewrite using list comprehension (all four lines can be written in a single line)